go/loopback/main.go

```diff
@@ -38,30 +37,29 @@ func Init(homeDir string, runModeStr string, serverURI string, accessGroupOverri
 			panic(err)
 		}
 
-		serviceLog = config.GetLogFile()
-
 		service := (service.NewService(g, false))
 		service.StartLoopbackServer()
 		service.G().SetService()
 
-		c = service.Contextified
+		serviceLog := config.GetLogFile()
+		logs := libkb.Logs{
+			Service: serviceLog,
+		}
+
+		logSendContext = libkb.LogSendContext{
+			Contextified: libkb.NewContextified(service.Contextified.G()),
+			Logs:         logs,
+		}
+
 		Reset()
 	})
 }
 
 // LogSend sends a log to kb
 func LogSend(uiLogPath string) (string, error) {
-	logs := libkb.Logs{
-		Service: serviceLog,
-		Desktop: uiLogPath,
-	}
-
-	logSendContext := libkb.LogSendContext{
-		Context: &c,
-		Logs:    logs,
-	}
+	logSendContext.Logs.Desktop = uiLogPath
 
-	return logSendContext.LogSend("", logs, 10000)
+	return logSendContext.LogSend("", 10000)
```

Comment:
Is there are reason you aren't including the status json for mobile?  And should 10000 be a parameter to this function?